src/components/_common/IconicButton/iconicButton.css

```diff
@@ -0,0 +1,12 @@
+.header-buttons {
+  padding: 6px;
+  border-radius: 50%;
+  font-size: 28px;
```

Comment:
this font-size also better to have in variables as "--icons-font-size" or so.

Reply:
resolved